charts/opentelemetry-operator/templates/clusterrolebinding.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.rbac.create }}
```

Comment:
Same questions here

Reply:
I would have the flag to switch between all or nothing. The issue in a multi tenant k8s cluster is that you cannot deploy ClusterRole at all.